test/csharp/synth-app/test.ts

```diff
@@ -8,13 +8,13 @@ import { TestDriver } from "../../test-helper";
 describe("csharp full integration test synth", () => {
   let driver: TestDriver;
 
-  beforeAll(() => {
+  beforeAll(async () => {
     driver = new TestDriver(__dirname)
-    driver.setupCsharpProject()
-  });
+    await driver.setupCsharpProject()
+  }, 180_000);
```

Comment:
would it be possible to define a global timeout for these `beforeAll` hooks?

Reply:
Yeah, we should probably override that globally somewhere 🙈